phoenicis-javafx/src/main/deploy/package/macosx/phoenicis-blue.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
For me it's a bit strange that we have this deploy now in phoenicis-javafx because all other deployment is done in phoenicis-dist. I think we should be consistent.

Reply:
Agreed but the plugin is not compatible with that :(

Reply:
I see. Nevertheless: Are the files in this directory even used anywhere?

Reply:
Yes, they are read by the plugin to generate the .app icon

Reply:
So these files are used by default? Just wondering because I can't find them anywhere in a pom.

Reply:
Yes

Reply:
Fine.